.github/workflows/automate.yaml

```
@@ -0,0 +1,15 @@
+name: Daily Data Retrieval Automation
+
+on:
+  schedule:
+    - cron: "0 0 */2 * *" # Runs every other day at 12AM
+
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run script
+        run: python main.py --days-needed 2 # Custom CLI Command
```

In order for this Python script to run on your friend's computer, what do they need to do first? They need to:
1. Install Python.
2. `pip install` the dependencies of `main.py`.
3. Set the required environment variables.

```suggestion
- name: setup python
        uses: actions/setup-python@v4
        with:
          python-version: '3.10'

      - name: Install Python packages
        run: |
          python -m pip install --upgrade pip
          pip install -r requirements.txt

      - name: Run script
        run: python main.py --days-needed 2 # Custom CLI Command
```

The `.github/workflows/automate.yaml` workflow is our friend, so let's treat it like one.

---

Resolved: https://github.com/acmcsufoss/acmOpportunities/pull/1/commits/ffde4fccbcd57a05155f221c96112227ecbfbc00

---

Resolved: https://github.com/acmcsufoss/acmOpportunities/pull/1/commits/3d5c6a78b32c8b15b7135f7dfa5c053f495d128d